salt/modules/docker.py

```diff
@@ -5971,7 +5971,6 @@ def sls_build(name, base='opensuse/python', mods=None, saltenv='base',
 
     # start a new container
     ret = create(image=base,
-                 name=name,
```

Comment:
The name is only used for generating a container with a name, we never use that name, we always use the id_ when referencing it

However in the `docker.image_present` state, we add the tag ':latest' to any image that does not already have a tag in it's `name` variable, and : is not a valid character in the name of a container, though we still want the tag in the name of the image.